ansible/playbooks/AdoptOpenJDK_AIX_Playbook/main.yml

```diff
@@ -15,28 +15,35 @@
     swap_size: 4096
 
   roles:
-    - aixfs
-    - syslogs
+    # AIX base requirements
     - X11
+    - openssl
+    # verify/install IBM compilers
     - xlc_v13
     - xlc_v16
-    - openssl
+    # AIX configuration
+    - aixfs
+    - syslogs
 
+    # yum based installation of additional software
     - yum
 
+    # users needed - wait until here to be sure bash is installed
+    - jenkins_user
+    - zeus_user
+
+    # additional OSS packages (needing python)
```

Comment:
Do these really need python? any is a java tool so shouldn't have any other prereqs on it

Reply:
This is what was in the original 'aix.yml' - I did not add it.

I will remove the comment test `(needing python)`.